go/tools/gopackagesdriver/json_packages_driver.go

```diff
@@ -23,7 +23,7 @@ type JSONPackagesDriver struct {
 	registry *PackageRegistry
 }
 
-func NewJSONPackagesDriver(jsonFiles []string, prf PathResolverFunc, bazelVersion string) (*JSONPackagesDriver, error) {
+func NewJSONPackagesDriver(jsonFiles []string, prf PathResolverFunc, bazelVersion bazelVersion) (*JSONPackagesDriver, error) {
```

Comment:
There is no function that returns a `bazelVersion` exposed in this package. This is implicitly making this constructor package private. If that's the intent, can you make the whole thing private `newJSONPackagesDriver`?

Reply:
Mind if I address that in a follow-up?

This is a `main` package so nothing can import it, but it has a lot of exported symbols. They all ought to be renamed.